content/academics.html

```diff
@@ -59,28 +122,130 @@ <h3> Favourite required courses </h3>
+    and <a href="https://uwflow.com/course/se380" target="_blank">SE 380 (Introduction to Feedback Control)</a> were the most disliked courses.
   </span>
-  Although SE 463 is just as disfavoured, it was a much easier course, which explains why 4A
-  was an easier term.
-  <br /><br />
-  It's important to note that courses that were disliked do not imply that
+  Most of the disliked courses were core courses, although some of them were also Linkage Electives (SMF 101, SPCOMM 100, SPCOMM 223)
+  <!-- Although SE 463 is just as disfavoured, it was a much easier course, which explains why 4A
+  was an easier term. -->
+  <!-- It's important to note that courses that were disliked do not imply that
   students find the content useless. Instead, there are a variety of factors
   such as personal interests, final grades, and assignments. For example, Logic and
   Computation (SE 212) is considered useful by 88% of students
   <a href="https://uwflow.com/course/se212" target="_blank">
-    according to UWFlow.</a>
+    according to UWFlow.</a> -->
 </p>
 <div class="half">
   <span class="graph-title">Least favourite mandatory courses</span>
   <div id="disliked-course"></div>
 </div>
+
+<h3> Largest Workload </h3>
+<p class="half">
+<span class="big">
+  Among core courses, <a href="https://uwflow.com/course/se350" target="_blank">SE 350 (Operating Systems)</a>
+  and <a href="https://uwflow.com/course/cs343" target="_blank">CS 343 (Concurrent and Parallel Programming)</a> were the courses with the largest workload. Among electives 
+  <a href="https://uwflow.com/course/cs444" target="_blank">CS 444 (Compiler Construction)</a>
+  was the course with the largest workload.
+</span>
+  Although these courses involved a high workload, most of them were also popular among respondents.
+</p>
+<div class="half second">
+  <span class="graph-title">Course with the largest workload</span>
+  <div id="largest-workload"></div>
+</div>
+
 <div class="vbx2"></div>
 
-<h2> Does X affect grades?</h2>
+<h2> Overloading </h2>
+<h3>Overloading & Academic Terms</h3>
+<p class="half">
+  <span class="big">
+  28 respondents overloaded at some point during their academic degree.
+  </span>
+  The 4A and 3A terms are the most popular term to overload. The SE curriculum encourages
+  students to overload in their 4A term to have a lighter term in 4B, which can be busy
+  with graduation traditions and FYDP work on top of courses.
+</p>
+<div class="half second">
+  <span class="graph-title"> Overloading vs Academic Terms</span>
+  <div id="overloading">
+  </div>
+</div>
 
-<h3> Attendance </h3>
+<h3>Overloading Reasons</h3>
+<p class="half">
+  <span class="big">
+  The most popular reasons to overload were to have a lighter term and to
+  take more electives than prescribed.
+</p>
+<div class="half second">
+  <span class="graph-title"> Reasons for Overloading</span>
+  <div id="overloading-reasons">
```

Comment:
in the preview, the x-axis (y-axis because we flipped it?) titles cut off; maybe we fix these issues in another pass?

![image](https://user-images.githubusercontent.com/11150920/120370139-34d98480-c2e2-11eb-8ae0-473173c54983.png)

Reply:
Yes, that was my plan